Samples/resource-owner-password-credential/Angular2/Angualar2-Client-ROPC/app/authorize/authorize-component.html

```diff
@@ -0,0 +1,81 @@
+﻿<ul *ngIf="!isLoggedin" class="nav navbar-nav navbar-right">
+    <li><button class="btn btn-link" (click)="mopen()">Login</button></li>
+</ul>
+<ul *ngIf="isLoggedin" class="nav navbar-nav navbar-right">
+    <li><button class="btn btn-link">Welcome {{userDetails.userName}}</button></li>
+    <li><button class="btn btn-link" (click)="logout()">Logout</button></li>
+</ul>
+<modal #myModal id="login-modal" tabindex="-1" role="dialog" style="display: none;">
+    <modal-header style="text-align:center">
+        <img class="img-circle" width="100" id="img_logo" src="http://bootsnipp.com/img/logo.jpg">
+    </modal-header>
+    <modal-body>
+        <form id="login-form" name="Loginform" [hidden]="!loginBool">
```

Comment:
I'm far from a bootstrap expert, but if you change this with `form-group` and `input-group` the UI will be layed out correctly. More info [here](http://getbootstrap.com/css/#forms)